ethereum/core/src/main/java/tech/pegasys/teku/core/ForkChoiceUtil.java

```diff
@@ -119,6 +119,26 @@ public static UInt64 compute_slots_since_epoch_start(UInt64 slot) {
     return roots;
   }
 
+  /**
+   * @param forkChoiceStrategy
+   * @param root
+   * @param startSlot
+   * @return every block root from root (inclusive) to start slot (exclusive) traversing the chain
+   *     backwards
+   */
```

Comment:
If we're going to add a java doc comment we should fill it all out. :)

I also feel like this should be some variant of `getAncestors` so that it has a similar name to the existing `getAncestors` method.  Maybe `getAncestorsOnFork`?